### ISSUE TITLE
Viewing Local Games in the Board

### ISSUE THREAD

**Issue by SamWheating** [BattlesnakeOfficial/rules#28]
The CLI has been really useful for quickly testing and iterating on a battlesnake, but it would be even better if we were able to output the match data to a file and then view it in a locally running [board](https://github.com/BattlesnakeOfficial/board) server. 

Is this something that would be possible given the current state of the CLI and the board?

I'm happy to help with this but may require some guidance.

**Comment by bvanvugt** [comment]
Hey @SamWheating, 

It's an interesting idea, and you're the second person to mention it in the past month. The current board needs to read stored GameStates from a websocket, so there's a few missing pieces to get this working. Not impossible though.

Have you used the ASCII board viewing option with the new CLI?

**Comment by SamWheating** [comment]
Yeah, the ASCII board viewing works but it isn't nearly as effective for watching / analyzing / rewinding gameplay.

Also, it can be impossible to tell the actual path of a snake or its head from a single ascii gameboard.

For example, in this board:

```
◦◦◦■⚕◦◦◦◦◦⚕
◦◦◦■■■◦◦■■◦
◦◦◦◦◦■◦◦■■⚕
◦◦◦◦◦■■■■■◦
◦◦◦◦◦◦◦◦◦■◦
◦◦◦◦◦⌀⌀⌀⌀■◦
◦◦⚕◦⚕⌀◦◦⌀■◦
◦◦◦◦◦⌀⌀⌀⌀■◦
◦◦⚕◦◦⌀⌀⌀⌀■◦
◦◦⌀⌀◦⌀⌀⌀⌀■◦
⚕◦⚕⌀⌀⌀⌀■■■◦
``` 


I believe there's multiple possible snake bodies which fit the shape of the `⌀` snake.
